test/conformance/kuttl/flags/standard/emit-events/05-script.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
+    if kubectl apply -f resource-fail.yaml 2>&1 | grep -q 'Error from server' 
```

Comment:
You should apply the resource that results in a violation (in audit mode) and grep the event with `PolicyViolation` type, and exit 1 if it's found, otherwise the test passes.